Fix typo (#782)
zcash/halo2: book/src/design/proving-system/vanishing.md

## Evaluating the polynomials

> At this point, all properties of the circuit have been committed to polynomials. The verifier now

Reviewer: ```suggestion
At this point, we have committed to all properties of the circuit. The verifier now
```

Author: @therealyingtong Applied!
